service/src/test/java/bio/terra/workspace/unit/TestTransitionalSerdes.java

```diff
@@ -0,0 +1,292 @@
+          {
+              "wsmControlledResourceFields": {
+                  "region": "us-central1",
+                  "applicationId": null,
+                  "managedBy": "MANAGED_BY_USER",
+                  "accessScope": "ACCESS_SCOPE_PRIVATE",
+                  "privateResourceState": "NOT_APPLICABLE",
+                  "assignedUser": null
+              },
+              "wsmResourceFields": [
+                  "bio.terra.workspace.service.resource.model.WsmResourceFields",
+                  {
+                      "workspaceId": "d7a7aa86-4a82-47b6-9c73-f416bb2eb7be",
+                      "lastUpdatedDate": null,
+                      "lastUpdatedByEmail": null,
+                      "createdDate": null,
+                      "createdByEmail": "dd@dd.com",
+                      "properties": [
+                          "com.google.common.collect.RegularImmutableMap",
+                          {
+                          }
+                      ],
+                      "resourceLineage": [
+                          "java.util.ArrayList",
+                          []
+                      ],
+                      "cloningInstructions": "COPY_NOTHING",
+                      "description": "fakeNotebook",
+                      "name": "fakenotebook",
+                      "resourceId": "814b272d-be51-4734-8057-2d150073fc8e"
+                  }
+              ],
+              "applicationId": null,
+              "region": "us-central1",
+              "lastUpdatedDate": null,
+              "lastUpdatedByEmail": null,
+              "createdDate": null,
+              "createdByEmail": "dd@dd.com",
+              "properties": [
+                  "com.google.common.collect.RegularImmutableMap",
+                  {
+                  }
+              ],
+              "resourceLineage": [
+                  "java.util.ArrayList",
+                  []
+              ],
+              "projectId": "projectId",
+              "location": "location",
+              "instanceId": "instanceid",
+              "managedBy": "MANAGED_BY_USER",
+              "accessScope": "ACCESS_SCOPE_PRIVATE",
+              "privateResourceState": "NOT_APPLICABLE",
+              "assignedUser": null,
+              "cloningInstructions": "COPY_NOTHING",
+              "description": "fakeNotebook",
+              "name": "fakenotebook",
+              "resourceId": "814b272d-be51-4734-8057-2d150073fc8e",
+              "workspaceId": "d7a7aa86-4a82-47b6-9c73-f416bb2eb7be"
+          }
+      ]
+      """;
+
+  @Test
+  public void testOldNotebookSerdes() throws Exception {
+    JobService.configureMapper();
+
+    FlightMap testMap = new FlightMap();
+    testMap.putRaw("foo", RESOURCE_NOTEBOOK_JSON);
+    ControlledAiNotebookInstanceResource resource =
+        testMap.get("foo", ControlledAiNotebookInstanceResource.class);
```

Comment:
should we test the deserialization for typeReference

Reply:
I don't think we need an additional test. The deserialization of something like `List<WsmResource>` uses the individual object deserialization inside.